src/test/authClass.logout.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { expect } from 'chai'
+import * as sinon from 'sinon'
+
+import { AuthService } from '../apiServices/AuthService'
+import { JwtTokensInterface } from '..'
+
+describe('logout', () => {
+    const tokens: JwtTokensInterface = {
+        access: 'access',
+        refresh: 'refresh',
+    }
+
+    let logoutUserStub: any
+    let AUTH
+    beforeEach(() => {
+        AUTH = new AuthService('user', 'password')
+        logoutUserStub = sinon.stub(AUTH, 'logout')
```

Comment:
It looks like we're not actually testing logout here. Because we are stubbing logout, we're not actually testing it's functionality. We want to stub the functions that logout is using, then call the real `AUTH.logout()`, and check that the stubs were called with the correct values.